src/clojurians_log/views.clj

```diff
@@ -16,6 +17,11 @@
   [messages ts]
   (some #(when (= (:message/ts %) ts) %) messages))
 
+(defn jsfile [path]
+  (when-let [f (io/file (io/resource (str "public" path)))]
+    (let [ts (.lastModified f)]
+      [:script
+       {:src (str path "?version=" ts)}])))
```

Comment:
In the lambdaisland code base we do something similar, but adding the content hash instead of the timestamp.

https://github.com/plexus/lambdaisland-app/blob/fbf6f776c420e3966e505f6447465e3deeea0346/src/lambdaisland/html/layout.clj#L13-L16

Maybe we should just go ahead and create a junk drawer library where we collect all these bits that are useful when doing server-side rendered applications.

Reply:
Yes, that's a nice idea, it could be helpful to other folks too!